src/components/DownloadAnytime.js

```diff
@@ -10,10 +10,10 @@ const DownloadAnytime = function ( props ) {
 
   return (
     <DownloadFile data={{client: props.client}}>
-      <Button type='button' color='teal' size='medium' className={'fixed'} id={'download_data_fixed'}>
+      <Button type='button' color='teal' size='medium' id={'download_data_fixed'}>
```

Comment:
I think the 'Reset' button will also be rotated and fixed. Could we add a rotated class that would do for both of those?